BUILD.md

```diff
@@ -2,144 +2,232 @@
-  `ssg-rhel7-ds.xml` is the Red Hat Enterprise Linux 7 source datastream.
-
-  The human readable HTML guide index files will be called
-  `ssg-${PRODUCT}-guide-index.html`. For example `ssg-rhel7-guide-index.html`.
-
-4. Discover the following:
- * A pretty prose guide **in ssg-rhel7-guide-index.html** containing practical, actionable information for administrators
- * A concise spreadsheet representation (potentially useful as the basis for an SRTM document) in **table-rhel7-nistrefs-stig-rhel7-disa.html**
- * Files that can be ingested by SCAP-compatible scanning tools, to enable automated checking:
-    * **ssg-rhel7-xccdf.xml**
-    * **ssg-rhel7-oval.xml**
-    * **ssg-rhel7-ds.xml**
-
-5. Install
-  * Custom location
-    `$ cd scap-security-guide/`
-    `$ cd build/`
-    `$ cmake ../`
-    `$ make -j4`
-    `$ make DESTDIR=/opt/absolute/path/to/ssg/ install`
-
-  * System-wide installation
-    `$ cd scap-security-guide/`
-    `$ cd build/`
-    `$ cmake ../`
-    `$ make -j4`
-    `$ make install`
-
-  * System-wide installation using ninja
-    `$ cd scap-security-guide/`
-    `$ cd build/`
-    `$ cmake -G Ninja ../`
-    `$ ninja-build`
-    `$ ninja-build install`
-
-### Building tarball, package and archive zipfile
-
-1. To build a tarball with all the source code.
-
-  `make package_source`
-
-2. To build a package with testing purposes.
-
-  * Disable any product you would not like to bundle in the package. For example:
-
-    `cmake -DSSG_PRODUCT_JBOSS_EAP5:BOOL=OFF../`
-
-  * Build the package.
-
-    `make package`
-
-  Currently, RPM and DEB packages are built.
-
-3. To build a zip file with all generated source data streams and kickstarts.
-
-  `make zipfile`
-
-## Using Docker
+### Installing build dependencies
+
+On *Red Hat Enterprise Linux* make sure the packages `cmake`, `openscap-utils`, `PyYAML`, `python-jinja2` and their dependencies are installed. We require version `1.0.8` or later of `openscap-utils`.
+```
+# yum install cmake openscap-utils PyYAML python-jinja2
+```
+
+On Fedora the package list is almost the same except for `python2-jinja2`:
+
+```
+# dnf install cmake openscap-utils PyYAML python2-jinja2
+```
+
+On *Ubuntu* and *Debian*, make sure the packages `libopenscap8`, `libxml2-utils`, `xsltproc`, `python-jinja2` and their dependencies are installed.
+```
+$ apt-get install cmake libopenscap8 libxml2-utils xsltproc python-jinja2
+```
+
+(optional) Install git if you want to clone the github repository to get the source code:
+```
+# yum install git
+```
+
+(optional) Install the ShellCheck package to perform fix script static analysis:
+```
+# yum install ShellCheck
+```
+
+(optional) Install the `ninja` build system if you want to use it instead of `make` for faster builds:
+
+```
+# yum install ninja-build
+```
+
+### Downloading the source code
+
+Download and extract a tarball from the [https://github.com/OpenSCAP/scap-security-guide/releases](list of releases):
```

Comment:
The HTML links are done differently in adoc.

Reply:
fixed